relentless/simulate/hoomd.py

```diff
@@ -401,15 +401,22 @@ class MinimizeEnergy(simulate.SimulationOperation):
         Force convergence criterion.
     max_iterations : int
         Maximum number of iterations to run the minimization.
-    dt : float
-        Maximum step size.
+    options : dict
+        Additional options for energy minimizer.
+
+    Raises
+    ------
+    ValueError
+        If a value for the maximum displacement is not provided.
```

Comment:
You should add a list of valid `options` to the documentation of this argument, and document what their type should be and what they do. You might be able to do that here with the argument by adding a bulleted list (Just be careful to render the docs and check it looks right), or else you could add a new section below.